M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -781,10 +844,11 @@ chooseGoodMinors(ZZ, ZZ, Matrix) := opts -> (howMany, minorSize, M1) -> (
     i := 0; -- how many times through the loop we go
     k := 0; -- how many actual minors we found
     j := 3; -- controlling when to reset the grevlex matrix
-    maxAttempts := 5*log_2(possibleMinors) + 10; --the absolute max number of attempts
+    maxAttempts := 10*log_1.3(possibleMinors) + 10; --the absolute max number of attempts
     R2 := reorderPolynomialRing(GRevLex, ambR);
     f := map(R2, ambR);
     sumMinors := trim ideal(sub(0, ambR));
+    --1/0;
```

Comment:
This doesn't look useful.

Reply:
I've removed the commented out --1/0s, throughout.